jax/_src/extend/ffi.py

```diff
@@ -185,10 +198,10 @@ def ffi_call(
   Args:
     target_name: the name of the XLA FFI custom call target that was registered
       using :func:`~jaxlib.xla_client.register_custom_call_target`.
-    result_shape_dtypes: an object, or sequence of objects, with ``shape`` and
-      ``dtype`` attributes which are expected to match the shape and dtype of
-      the custom call output or outputs. :class:`~jax.ShapeDtypeStruct` is often
-      used to define the elements of ``result_shape_dtypes``.
+    results: a ``jax.core.Value`` or sequence thereof that are of the same type
+      and shape as the custom call output or outputs. For array-like outputs
+      :class:`~jax.ShapeDtypeStruct` is often used to define the elements of
+      ``results``.
```

Comment:
`jax.core` isn't typically considered a public API so it shouldn't be included in the docstring. How about just using the existing text and then adding a sentence about tokens?